commands/doit.go

```diff
@@ -343,4 +392,35 @@ func CmdBuilder(parent *Command, cr CmdRunner, cliText, desc string, out io.Writ
 	}
 
 	return c
+
+}
+
+func writeConfig() error {
+	f, err := cfgFileWriter()
+	if err != nil {
+		return err
+	}
+
+	defer f.Close()
+
+	b, err := yaml.Marshal(viper.AllSettings())
+	if err != nil {
+		return errors.New("unable to encode configuration to YAML format")
+	}
+
+	_, err = f.Write(b)
+	if err != nil {
+		return errors.New("unable to write configuration")
+	}
+
+	return nil
+}
+
+func defaultConfigFileWriter() (io.WriteCloser, error) {
+	f, err := os.Create(cfgFile)
```

Comment:
since this contains API tokens, you might want to reduce the permissions to something like `0640`